### PR TITLE
chore: disable unlimited associations

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Set default to `false`.

### CODE REVIEW

**Review by david-bakin-sl — APPROVED** [review]
LGTM

**Review by thomas-swirlds-labs — DISMISSED** [review]
Looks good - Thanks @tinker-michaelj 😎

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM! Thanks @tinker-michaelj

**Review by Nana-EC — DISMISSED** [review]

File: hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/HevmTransactionFactory.java [hunk 1]
```diff
@@ -326,8 +326,8 @@ private void assertValidCreation(@NonNull final ContractCreateTransactionBody bo
         validateTrue(body.gas() <= contractsConfig.maxGasPerSec(), MAX_GAS_LIMIT_EXCEEDED);
         final var usesUnsupportedAutoAssociations =
                 body.maxAutomaticTokenAssociations() > 0 && !contractsConfig.allowAutoAssociations();
-        final var usesInvalidAutoAssociations = body.maxAutomaticTokenAssociations() < UNLIMITED_AUTOMATIC_ASSOCIATIONS
-                && entitiesConfig.unlimitedAutoAssociationsEnabled();
+        final var usesInvalidAutoAssociations = body.maxAutomaticTokenAssociations()
+                < (entitiesConfig.unlimitedAutoAssociationsEnabled() ? UNLIMITED_AUTOMATIC_ASSOCIATIONS : 0);
```
Comment (david-bakin-sl) [inline hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/HevmTransactionFactory.java:330]: Is this encoding the knowledge that `.maxAutomaticTokenAssociations` must be `-1` for unlimited?  It'd be kind of nicer to be explicit.  IMO.

**Review by david-bakin-sl — APPROVED** [review]
LGTM

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM! It seems like this feature is very tightly coupled with lot of tests 😓 . tyvm @tinker-michaelj

**Comment by david-bakin-sl** [comment]
PR description seems wrong now if for no other reason than there's a bunch of (opaque) fee schedule json changes in here.  What's really happening besides changing that one `true` -> `false` over in `EntitiesConfig.java`?

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**